core/src/main/java/hudson/model/Run.java

```diff
@@ -58,18 +58,8 @@
 import hudson.util.LogTaskListener;
 import hudson.util.ProcessTree;
 import hudson.util.XStream2;
-import java.io.BufferedReader;
-import java.io.ByteArrayInputStream;
-import java.io.File;
-import java.io.FileInputStream;
-import java.io.FileOutputStream;
-import java.io.IOException;
-import java.io.InputStream;
-import java.io.InputStreamReader;
-import java.io.OutputStream;
-import java.io.PrintWriter;
-import java.io.Reader;
-import java.io.StringWriter;
+
+import java.io.*;
```

Comment:
Please do not do it in the production code. We have so many internal classes with "common" names and interfaces. It may cause implicit ambiguous behavior